uiplib/gui/gallery.py

```diff
@@ -62,3 +62,23 @@ def blur_image(self):
         """Apply blur to the chosen image."""
         show = self.image.blur(self.slider.get())
         self.show_image(show)
+
+    def remove(self, object):
+        """Remove object from gallery if exists."""
+        if object:
+        object.pack_forget()
```

Comment:
It was written in an unnecessary hurry. 
Keep on reviewing. After @lohani2280 's commit is merged I will fix all the gaps and polish it properly.